packages/frontend/app/components/app/off-canvas/-page.ts

```diff
@@ -45,8 +46,16 @@ export const page = create({
     search: fillable('[data-test-sidebar-search]'),
     searchInfo: text('[data-test-search-info]'),
 
+    selectContactsTab: clickable('[data-test-tab-contacts]'),
+    selectChannelsTab: clickable('[data-test-tab-channels]'),
+    selectActionsTab: clickable('[data-test-tab-actions]'),
+
+    //TODO: turn into single identifier to test which text appears.
```

Comment:
what does this mean?

Reply:
Realized I can in fact implement this with just extra use of the feature flag.